ui/apps/platform/src/Components/visuals/SunburstDetailSection.js

```diff
@@ -79,28 +78,19 @@ class SunburstDetailSection extends Component {
                                 {link && (
                                     <Link
                                         title={text}
-                                        className={`underline leading-normal flex w-full word-break ${
-                                            className ?? ''
-                                        }`}
-                                        style={color ? { color } : null}
+                                        className="leading-normal flex w-full word-break"
                                         to={link}
                                     >
                                         <Truncate lines={6} ellipsis={<>...</>}>
                                             {text}
                                         </Truncate>
                                     </Link>
                                 )}
-                                <span
-                                    className="flex w-full word-break leading-tight"
-                                    style={color ? { color } : null}
-                                >
+                                <span className="flex w-full word-break leading-tight">
                                     <Truncate lines={4} ellipsis={<>...</>}>
                                         {!link && text}
                                     </Truncate>
                                 </span>
-                                {selectedDatum && units !== 'percentage' && (
-                                    <span style={{ color: labelColor }}>{labelValue}</span>
-                                )}
```

Comment:
Was this an obsolete prop?

Reply:
Good question. In the 4 components that render `Sunburst` element:
* `labelValue` property has no occurrences.
* `labelColor` property has occurrence only in `CvesByCvssScore` component.